src/game.c

```diff
@@ -43,9 +43,147 @@ void sub_80191A4(void);
 
 extern const u16 gLevelSongs[];
 
-extern const u16 gUnknown_080D5964[][2];
-// TODO not TaskMain use an inline type
-extern const TaskMain gUnknown_080D57DC[];
+void sub_801AF14(void);
+void sub_801AFA4(void);
+void sub_801B03C(void);
+void sub_801B0D4(void);
+void sub_801B180(void);
+void sub_801B22C(void);
+void sub_801B324(void);
+void sub_801B41C(void);
+void sub_801B4EC(void);
+void sub_801B5BC(void);
+void sub_801B7BC(void);
+void sub_801B83C(void);
+void sub_801B8A8(void);
+void sub_801B914(void);
+void sub_801B980(void);
+void sub_801B9EC(void);
+void sub_801BA58(void);
+void sub_801BAC4(void);
+void sub_801BB30(void);
+void sub_801BB9C(void);
+void sub_801BC88(void);
+void sub_801BC1C(void);
+void sub_801BD08(void);
+void sub_801BD74(void);
+void sub_801BDE0(void);
+void sub_801BE4C(void);
+void sub_801BEB8(void);
+void sub_801BF24(void);
+const VoidFn gUnknown_080D57DC[]
+    = { sub_801AF14, sub_801B7BC, sub_801B83C, sub_801BF24, // Anti-Formatting
+        sub_801B8A8, sub_801B914, sub_801B980, sub_801BF24, //
+        sub_801AFA4, sub_801B03C, sub_801B9EC, sub_801BF24, //
+        sub_801B0D4, sub_801B180, sub_801BA58, sub_801BF24, //
+        sub_801BAC4, sub_801BB30, sub_801BB9C, sub_801BF24, //
+        sub_801B22C, sub_801B324, sub_801BC1C, sub_801BF24, //
+        sub_801B41C, sub_801B4EC, sub_801B5BC, sub_801BF24, //
+        sub_801BC88, sub_801BD08, sub_801BD74, sub_801BDE0, //
+        sub_801BE4C, sub_801BEB8 };
+
+asm(" .global gUnknown_080D5864\n"
```

Comment:
This is background templates `extern const Background gUnknown_080D5864[4]`

Reply:
I can write a script to extract if you want?

Reply:
Would be great to have some extraction scripts, especially for these, if possible:
https://github.com/freshollie/sa2/blob/d068de31cca0d98a21720af59f4600210321a22c/data/map_data.s#L254